app/scripts/background.js

```diff
@@ -676,10 +676,8 @@ export function setupController(
   //
   // User Interface setup
   //
```

Comment:
Why do we no longer need to wait for `initApprovals()` to call `updateBadge()`?

Reply:
The previous implementation of `initApprovals` wouldn't resolve until the requests were approved, meaning the badge number would be zero at that point anyway.

Plus I noticed we already listened to the `updateBadge` event from the approval controller below anyway, meaning the badge is already reacting to the creation of the approval requests.